modules/cnv_creating_template_wizard_web.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv_users_guide.adoc
+
+[[cnv-creating-template-wizard-web]]
+== Creating a virtual machine template with the interactive wizard
+
+Virtual machine templates are an easy way to create multiple virtual machines with similar configuration. After a template is created, reference the template when xref:cnv-creating-vm-wizard-web[creating virtual machines].
+
+The web console features an interactive wizard that guides you through xref:cnv-template-wizard-fields-web[*Basic Settings*], xref:cnv-networking-wizard-fields-web[*Networking*], and xref:cnv-storage-wizard-fields-web[*Storage*] screens to simplify the process of creating virtual machine templates. All required fields are marked by a `*`. The wizard prevents you from moving to the next screen until the required fields have been completed.
+
+NICs and storage disks can be created and attached to virtual machines after they have been created. 
+
+.*Bootable Disk*
+
+If either `URL` or `Container` are selected as the *Provision Source* in the xref:cnv-template-wizard-fields-web[*Basic Settings*] screen, a `rootdisk` disk is created and attached to virtual machines as the *Bootable Disk*. You cannot remove or modify the `rootdisk`.
```

Comment:
@aburdenthehand Hm.. This wording is not correct in this case. One can't changes the Bootable disk, which is rootdisk to something else. But the rootdisk itself can be modified, if the provision source is URL. One can change its size and the Storage class. For Container source, one can modify only a name of the rootdisk, and Bootable disk will refer accordingly.

Reply:
Updated to "You can modify the `rootdisk` but you cannot remove it. "
also added substep to the Storage step in procedure: ".. (Optional) Click on a disk to modify available fields. Click the &#10003; button to save the update."

We could specifically call out the difference in what can be updated between container- and URL-based rootdisks but it makes this para convoluted for what should be straightforward to anyone using the wizard.